### PR TITLE
[EDU-3571] feat: add login command reference in pt

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue
[EDU-3571](https://aziontech.atlassian.net/browse/EDU-3571)

Changes

- Add login command reference

[EDU-3571]: https://aziontech.atlassian.net/browse/EDU-3571?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli-alpha/login/login.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Azion CLI login
+description: Faça login na sua conta da Azion e salve um personal token localmente para autorizar comandos CLI.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli-alpha/login/login.mdx:3]:
```suggestion
description: Faça login na sua conta da Azion e salve um personal token localmente para autorizar comandos da CLI.
```

**Review by MarianaAguilera — APPROVED** [review]
apenas uma sugestão

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli-alpha/login/login.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+meta_tags: cli, login, autenticação
+namespace: documentation_cli_login
+menu_namespace: cliMenuAlpha
+permalink: /documentacao/produtos/cli/login/
+---
+
+Faça login na sua conta da Azion e salve um personal token localmente para autorizar comandos na Azion CLI.
+
+:::note
+No caso de um ou mais campos obrigatórios não serem informados através das flags específicas, uma mensagem interativa será exibida, solicitando essas informações.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli-alpha/login/login.mdx:13]:
```suggestion
Se um ou mais campos obrigatórios não forem informados através das flags específicas, uma mensagem interativa será exibida, solicitando essas informações.
```

**Review by hannah-sk — APPROVED** [review]
sugeri um ajuste

**Pull request merged**